pkg/controllers/datastoresyncer/datastoresyncer.go

```diff
@@ -83,7 +83,7 @@ func (d *DatastoreSyncer) ensureExclusiveEndpoint() {
 
 	for _, endpoint := range endpoints {
 		if !util.CompareEndpointSpec(endpoint.Spec, d.localEndpoint.Spec) {
-			endpointCrdName, err := util.GetEndpointCRDName(endpoint)
+			endpointCrdName, err := util.GetEndpointCRDName(&endpoint)
```

Comment:
This also looks like it was pulled in from another patch, is that right?

Reply:
Sorry for the noise, this should not be here, bad rebase.